tests/trace/test_evaluations.py

```diff
@@ -420,7 +431,14 @@ async def test_evaluation_data_topology(client):
 
     # Prediction
     assert predict_call.output == model_output
-    assert predict_call.summary == predict_usage
+    summary = {
+        **predict_call.summary,
+        "weave": {
+            **predict_call.summary["weave"],
+            "feedback": [],
+        },
+    }
+    assert summary == with_empty_feedback(predict_usage)
```

Comment:
why are you doing this?

Reply:
I am now requesting feedback which is used below, but don't want to make assertions about it here. Stripping the feedback makes that nice. I cleaned up the code to make it cleaner.